### PR TITLE
[yandex-maps] initial version

### PR DESCRIPTION
**Pull request by delagen**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by Delagen** [comment]
This is for https://tech.yandex.ru/maps/doc/jsapi/ so no NPM package exists

File: types/yandex-maps/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3043 @@
...
+
+		geometry: IGeometry | null;
+		indices: ArrayBuffer;
+		vertices: ArrayBuffer;
+
+		getOverlay(): Promise<IOverlay | null>;
+
+		getOverlaySync(): IOverlay | null;
+
+		getParent(): any | IControlParent;
```
Comment (minestarks) [inline types/yandex-maps/index.d.ts]: I saw many other properties like this, so I'm just commenting on this one:

Using `any` in union types is not a good idea. `any | IControlParent` is still the same type as `any`. Consider just using `any`, just `IControlParent`, or maybe `IControlParent | object` depending on what you're trying to do?

**Review by minestarks — CHANGES_REQUESTED** [review]
Please address comment about "T | any" props

File: types/yandex-maps/index.d.ts (hunk 1 above)
Comment (Delagen) [reply types/yandex-maps/index.d.ts]: Sad for this. This is due WebStorm by default implements interface of T|null to T|any. Fixed.

**Review by minestarks — APPROVED** [review]

**Comment by minestarks** [comment]
Thank you!

**Pull request merged**